rfcs/text/0000-kubernetes.md

```diff
@@ -0,0 +1,125 @@
+# 0000: Kubernetes schema creation
```

Comment:
Kubernetes will definitely be the largest (only? 😂 ) container orchestration project considered in this discussion, but ECS prefers concept names over project or tool names. 

I propose using "orchestrator" over Kubernetes. Thoughts?

Also, the terms "fields" or "field set" would better describe the intent here, instead of "schema". In its entirety, ECS is the schema, and the fields are organized into field sets within that schema.

Reply:
Orchestrator sounds reasonable to me - my only reservation is that there are several concepts in Kubernetes that would be useful to express as fields, but I'm not sure how extensible these are to other orchestrator architectures - I am most familiar with Kubernetes. 

If we are going to try and make this generic I think it would be useful to get some input from someone who has experience with Nomad - not sure if there are any other container orchestration engines that we should be thinking about.

I will go with changing this to 'orchestrator' for now, and we can see how we get on I guess!